open-source-services.md

```diff
@@ -73,19 +101,12 @@ However, if a typical team member operates their own instance of the service in
 It is a powerful, laudable goal for others to be able to operate an Open Source service, either by operating an instance on their own, or participating in operations of a common instance.
 [Operate First](https://www.operate-first.cloud/) is a project that enables this, and even provides infrastructure.
 
-### Q: Must all dependencies for a service be Open Source too?
-
-**A:** It is _not required_ that all dependencies of a service are Open Source, or have Open Source alternatives.
-For example, a service may interact with a proprietary authentication system, or run on a proprietary platform.
-
-The requirements together serve to enable others to change the service code so that its proprietary dependencies can be replaced with Open Source ones, should that be desired.
-
```

Comment:
This was moved above, and the FAQ question was changed to include assets. While I'm fine with moving the two questions one after another, I believe the topic of assets and dependencies should have separate questions even if the answers refer to one another in subtle ways.